api/v1alpha1/enterprisecontractpolicy_types.go

```diff
@@ -23,46 +23,46 @@ import (
 // Important: Run "make" to regenerate code after modifying this file
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// EnterpriseContractPolicySpec represents the desired state of EnterpriseContractPolicy
+// EnterpriseContractPolicySpec is used to configure the Enterprise Contract Policy
 type EnterpriseContractPolicySpec struct {
-	// Description text describing the the policy or it's intended use
+	// Arbitrary description of the policy or it's intended use
 	// +optional
-	Description *string `json:"description"`
-	// Sources is list of policy sources
+	Description *string `json:"description,omitempty"`
+	// One or more policy sources
 	// +kubebuilder:validation:MinItems:=1
 	Sources []PolicySource `json:"sources"`
-	// Authorization defines a release approval
+	// Authorization defines per component release approvals
 	// +optional
-	Authorization Authorization `json:"authorization"`
+	Authorization *Authorization `json:"authorization,omitempty"`
```

Comment:
do we need a corresponding PR in ec-cli to update to use pointers?

Reply:
Yeah, once this gets merged...